tmeister/static/spec/helpers/mock-globals.js

```diff
@@ -7,6 +7,9 @@ mockGlobals([
 			getItem() {},
 		}
 	},
+	document: {
+		getElementById() {}
+	},
```

Comment:
Coverage is broken with a document dependency. I was attempting to get it fixed but decided I don't care much about coverage. Maybe we'll fix it later :)